meetings/eng/2022.md

```diff
@@ -1,5 +1,102 @@
 # 2022 Meeting Notes
 
+## 2022-05
+
+- Facilitator: Sarah
+- Notetaker: Erik, Jim
+
+### Job hire update/decision making/discussion
+
+- [Proposal document is here](https://docs.google.com/document/d/1PHpLJniwOIWXrhUcEiFhnOwPVQC9NSkoyHQeDss4udg/edit?usp=sharing)
+  - Brief description of current candidates
+  - Describe suggestion to make an offer
+  - Question: Any objections to making the offer?
+  - Discuss potential second hire
+  - Question: Worth pursuing if we have budget?
```

Comment:
```suggestion
```

I think we should strike these too, since we don't know if it is going to happen and this might be classified as sensitive info